chain/sync.go

```diff
@@ -1172,7 +1172,7 @@ func (syncer *Syncer) checkBlockMessages(ctx context.Context, b *types.FullBlock
 	}
 
 	// Finally, flush.
-	return vm.Copy(ctx, tmpbs, syncer.store.Blockstore(), mrcid)
```

Comment:
Pretty sure this should be the chain blockstore

Reply:
(and my comment [here](https://github.com/filecoin-project/lotus/pull/5695#discussion_r584992695) was wrong, I just looked at `vm.Copy` and thought this must be touching the state)

Reply:
I knew I was right the first time :-)